commands/members/index.js

```diff
@@ -1,37 +1,67 @@
 'use strict'
 
+let _ = require('lodash')
 let cli = require('heroku-cli-util')
 let co = require('co')
-let _ = require('lodash')
+let Utils = require('../../lib/utils')
 
 function * run (context, heroku) {
-  let members = yield heroku.get(`/organizations/${context.org}/members`)
-  members = _.sortBy(members, 'email')
+  let orgInfo = yield Utils.orgInfo(context, heroku)
+  let groupName = context.org || context.flags.team
+  let teamInvites = []
+
+  if (orgInfo.type === 'team') {
+    let orgFeatures = yield heroku.get(`/organizations/${groupName}/features`)
+
+    if (orgFeatures.filter(of => of.name === 'team-invite-acceptance').length) {
+      teamInvites = yield heroku.request({
+        headers: {
+          Accept: 'application/vnd.heroku+json; version=3.team-invitations'
+        },
+        method: 'GET',
+        path: `/organizations/${groupName}/invitations`
+      })
+      teamInvites = _.map(teamInvites, function (invite) {
+        return {email: invite.user.email, role: invite.role, status: 'pending'}
+      })
+    }
+  }
+
+  let members = yield heroku.get(`/organizations/${groupName}/members`)
+  // Set status '' to all existing members
+  _.map(members, (member) => { member.status = '' })
+  members = _.sortBy(_.union(members, teamInvites), 'email')
   if (context.flags.role) members = members.filter(m => m.role === context.flags.role)
+  if (context.flags.pending) members = members.filter(m => m.status === 'pending')
   if (context.flags.json) {
-    cli.log(JSON.stringify(members, null, 2))
+    cli.log(JSON.stringify(members, null, 3))
```

Comment:
This seems a bit disingenuous. Isn't the `json` flag a representation of the API? I realize we have two types of things here, but perhaps we should serialize them as those two things rather than the union of the two of them.